db/dynamodb-global-table.yml

```diff
@@ -0,0 +1,337 @@
+    Default: PAY_PER_REQUEST
+  ReadCapacitySeed:
+    Type: Number
+    Default: 1
+    Description: Initial read capacity when creating or switching to provisioned billing
+  ReadCapacityMin:
+    Type: Number
+    Default: 1
+    Description: Only used under provisioned billing
+  ReadCapacityMax:
+    Type: Number
+    Default: 1
+    Description: Only used under provisioned billing
+  ReadUtilizationScalingTarget:
+    Type: Number
+    Default: 70
+    Description: Only used under provisioned billing
+  WriteCapacitySeed:
+    Type: Number
+    Default: 1
+    Description: Initial read capacity when creating or switching to provisioned billing
+  WriteCapacityMin:
+    Type: Number
+    Default: 1
+    Description: Only used under provisioned billing
+  WriteCapacityMax:
+    Type: Number
+    Default: 1
+    Description: Only used under provisioned billing
+  WriteUtilizationScalingTarget:
+    Type: Number
+    Default: 70
+    Description: Only used under provisioned billing
+  AutoScalingCooldown:
+    Type: Number
+    Default: 60
+    Description: Only used under provisioned billing
+
+Conditions:
+  HasExpirationField: !Not [!Equals [!Ref ExpirationField, ""]]
+  HasSecondaryRegion: !Not [!Equals [!Ref Region2, ""]]
+  HasTertiaryRegion: !Not [!Equals [!Ref Region3, ""]]
+  HasForeignAccount: !Not [!Equals [!Ref ForeignAccountId, ""]]
+  HasForeignRole: !Not [!Equals [!Ref ForeignRoleName, ""]]
+  HasProvisionedBilling: !Equals [!Ref BillingMode, "PROVISIONED"]
+
+Mappings:
+  DataTypesMap:
+    String:
+      Symbol: S
+    Numeric:
+      Symbol: N
+    Binary:
+      Symbol: B
+
+Resources:
+  DynamoDBTable:
+    Type: AWS::DynamoDB::GlobalTable
+    Properties:
+      AttributeDefinitions:
+        - AttributeName: !Ref PrimaryKey
+          AttributeType: !FindInMap [DataTypesMap, !Ref PrimaryType, Symbol]
+      BillingMode: !Ref BillingMode
+      KeySchema:
+        - AttributeName: !Ref PrimaryKey
+          KeyType: HASH
+      Replicas:
+        - ReadProvisionedThroughputSettings: !If
+            - HasProvisionedBilling
+            - ReadCapacityAutoScalingSettings:
+                MaxCapacity: !Ref ReadCapacityMax
+                MinCapacity: !Ref ReadCapacityMin
+                SeedCapacity: !Ref ReadCapacitySeed
+                TargetTrackingScalingPolicyConfiguration:
+                  ScaleInCooldown: !Ref AutoScalingCooldown
+                  ScaleOutCooldown: !Ref AutoScalingCooldown
+                  TargetValue: !Ref ReadUtilizationScalingTarget
+            - !Ref AWS::NoValue
+          Region: !Ref AWS::Region
+          Tags:
+            - { Key: prx:meta:tagging-version, Value: "2021-04-07" }
+            - { Key: prx:cloudformation:stack-name, Value: !Ref AWS::StackName }
+            - { Key: prx:cloudformation:stack-id, Value: !Ref AWS::StackId }
+            - { Key: prx:ops:environment, Value: !Ref EnvironmentType }
+            - { Key: prx:dev:family, Value: !Ref FamilyTag }
```

Comment:
I don't imagine us using this template for much else, so probably fine for now, but the `family` tag should be optional.